### DISCUSSION TITLE
Project progress

### DISCUSSION THREAD

**Discussion by ian-tiggeman** [ian-tiggeman/T24S2T1EXTG08#2] [General]
Cara,
I have now completed Section B of this project, so we can now start Section C (individual work).
Let me know if you have any questions.
Are you happy to do the Mortgage Calculator feature?
Regards,
Ian

**Comment by DanyaoWang-001167576** [comment]
Hi Ian,
Thanks for setting up the repository. I’m happy to do the Mortgage Calculator feature :)
Regards,
Cara

**Reply by ian-tiggeman** [reply]
Excellent. Thanks Cara.
I will obviously work on the Currency Converter feature.
Please remember to send me your public key (via email) so you can begin to collaborate on GitHub.
